client/components/Story.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import styled from 'styled-components';
+import { StyledDesc, NavHeader, SectionHeader, Caption, Img } from '../StyledDiv';
+
+const Story = (props) => {
+  const {story} = props;
```

Comment:
not sure I understand these two lines. I'd be interested to chat about them later.

Reply:
Now that you point it out, I actually just refactored it to consolidate those lines into one: const Story = ({story}) => {